datumaro/util/test_utils.py

```diff
@@ -115,26 +116,34 @@ def compare_categories(test, expected, actual):
 IGNORE_ALL = "*"
 
 
-def compare_annotations(expected, actual, ignored_attrs=None):
-    if not ignored_attrs:
+def compare_annotations(expected: Annotation, actual: Annotation, ignored_attrs=None):
+    is_skeleton = expected.type == actual.type == AnnotationType.skeleton
+    if not ignored_attrs and not is_skeleton:
         return expected == actual
 
-    a_attr = expected.attributes
-    b_attr = actual.attributes
+    ignored_attrs = ignored_attrs or {}
```

Comment:
Isn't `ignored_attrs` supposed to be a list?

Reply:
It's defined as `Union[None, Literal["*"], Collection[str]]` in `compare_datasets`, so it can be any collection. Replaced `{}` with `[]` anyway